src/store.js

```diff
@@ -25,6 +25,8 @@ export class Store {
       strict = false
     } = options
 
+    this.applyMixins(options)
```

Comment:
This applies mixins only root options. We should even apply them for all child modules.